Minor proofreading proposals for docs

masonmark opened the pull request:
Hello, this PR is just a somewhat random set of very trivial proposed changes that I compiled during my first read-through of the docs. 

Just minor stuff like apostrophies, capitalization, and adding optional commas and simplifying phrases for clarity and to help non-native English speakers.

Congrats on the 1.0! 🙂

masonmark committed:
Add minor proofreading corrections

masonmark committed:
Small proofreading changes

masonmark committed:
Small proofreading changes

masonmark committed:
Small proofreading changes

masonmark committed:
Small proofreading changes

masonmark committed:
Small proofreading changes

masonmark committed:
Small proofreading changes

amrbashir commented on docs/guides/debugging/application.md:
```diff
@@ -67,7 +67,7 @@ tauri::Builder::default()
 ### Using the Inspector in Production
 
-By default the inspector is only enabled in development and debug builds unless you enable it with a Cargo feature.
+By default the inspector is only enabled for development and debug builds unless you enable it with a Cargo feature.
```
I feel like "in" conveys the meaning better, but I am not a native speaker so I am just double checking

masonmark replied:
👍 Yeah, I think that is valid.

amrbashir reviewed (changes requested):
Thank you, this is great

masonmark committed:
in → for^Win

The choice of "in" vs "for" is a little tricky even for native English speaker: the enablement is in the build process, not the resulting app, but on the other hand the inspector is displayed **in** the actual resulting application's UI. I have no objection to reverting my proposal "for" and keeping this "in". 👍

lorenzolewis commented:
@amrbashir does this look good to you now?

amrbashir reviewed (approved):
Looks good, thanks again
